vlm/src/main/scala/geotrellis/contrib/vlm/LayoutTileSource.scala

```diff
@@ -119,7 +119,7 @@ class LayoutTileSource(val source: RasterSource, val layout: LayoutDefinition) e
     readAll(keys.toIterator)
 
   /** Set of keys that can be read from this tile source */
-  def keys(): Set[SpatialKey] = {
```

Comment:
Not that I'm against it, but why the switch?
Should this be a default style we choose for 0-argument methods ?